lib/utils/configParser.js

```diff
@@ -38,3 +38,9 @@ module.exports.ChangeStartPage = function(hostedPage, configXML) {
     }), "utf-8");
     return filename;
 };
+
+module.exports.GetStartPage = function(configXML) {
+    var parsedConfigXML = parseXml(configXML);
+    var contentTag = parsedConfigXML.find('content[@src]');
```

Comment:
This will work well in the happy path - but there is also an error condition where there is no content tag. It's likely rare - but something that would be hard to debug and will result in a random failure. I'm a big fan of defensive coding - but it's OK at this point to let this be this way.